extensions/amp-consent/validator-amp-consent.protoascii

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-consent
+  html_format: AMP
+  html_format: EXPERIMENTAL
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-consent"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+    requires_usage: GRANDFATHERED
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # amp-consent (json)
```

Comment:
Since `<amp-consent>` is `unique`, should this also be `unique` or can runtime handle multiple json scripts as children of `<amp-consent>`.

Reply:
One question: how can I specify that there should be at least one and only one json script? Is `unique` enough? Note `<amp-consent>` could have multiple other child nodes.

Reply:
Can any of the other multiple child nodes be a `<script type="application/json>`? If not then `unique: true` is what you want.

Reply:
If this is to be tied expilcitly to `<amp-consent>` tag and only one exists then it needs:
`satisfies: "amp-consent extension .json script"` and `unique: true`.
Then `<amp-consent>` tag would need:
`requires: "amp-consent extension .json script"`

Reply:
Thanks! Changed.